src/components/ProcessedContext/ProcessedContext.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+interface ContextValues {
+  error: ErrorTypes,
+  setError: React.Dispatch<React.SetStateAction<ErrorTypes>>,
+  isAdding: boolean,
+  setIsAdding: React.Dispatch<React.SetStateAction<boolean>>,
+  processedTodoIds: number[],
+  setProcessedTodoIds: React.Dispatch<React.SetStateAction<number[]>>,
+}
+
+export const ProcessedContext = React.createContext<ContextValues>({
+  error: ErrorTypes.GET,
+  setError: () => {},
+  isAdding: false,
+  setIsAdding: () => {},
+  processedTodoIds: [],
+  setProcessedTodoIds: () => {},
+});
```

Comment:
I think that it is better to split this context to ErrorContext and Processed Context, because they sre responsible for different handles

Reply:
I didn't want to create many contexts (( 
But maybe you're right)
Let's see what the mentor has to say?)

Reply:
Yes, it is interesting, what mentor will say)